sampleFiles/sample-schema.json

```diff
@@ -1946,6 +1946,864 @@
       ],
       "name": "ARGO Clinical Submission",
       "version": "2.0"
+    },
+    {
```

Comment:
I added a new version cause I wasn't sure which one we should add to, I think this makes sense no?

Reply:
you can just add it to 1.0 the others are not used by default